e2e/terraform/packer/ubuntu-jammy-amd64/cni_args.conflist

```diff
@@ -0,0 +1,9 @@
+{
+	"cniVersion": "1.0.0",
+	"name": "cni_args",
+	"plugins": [
+		{
+		    "type": "cni_args.sh"
+		}
+	]
+}
```

Comment:
This file is missing a EOL at the end of the file, which will mess up various unix tools (ex. try running `wc -l` on this file and you'll see it'll only have 8 lines!). You should be able to fix-on-save in your editor.